verilog/tools/formatter/verilog_format.cc

```diff
@@ -126,10 +128,13 @@ static std::ostream& FileMsg(absl::string_view filename) {
 }
 
 static bool formatOneFile(absl::string_view filename,
-                          const LineNumberSet& lines_to_format) {
+                          const LineNumberSet& lines_to_format,
+                          bool* lines_changed) {
```

Comment:
maybe instead of `lines_changed`, call it `any_change` ? Lines changed sounds like a number of lines we'd get back, while `any_change` more emphasizes the boolean nature of things (also below in `main()`)

Reply:
Done.